ProffieOS.ino

```diff
@@ -18,8 +18,16 @@
  along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-// You can have multiple configuration files, and specify which one
-// to use here.
+/*
+---------------------------------------------------------------------
```

Comment:
How about this style?

```
/*---------------------------*\
| text in goes in here        |
\*---------------------------*/

```

I also like this style:

```
/*\
| |  Some text goes here
\*/

```

Reply:
ooo fancy, nice! 
How about the Version included as shown so users can confirm the OS?
I always throw that in my personal version.
Although, the master branch isn't really......